README.md

```diff
@@ -1,8 +1,20 @@
-# React + Vite
+# Northcoders News App
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+Welcome to my NC-News-App. This is a project I've been working on to learn how to use React to create a web app for my NC_News_API. It allows the user to navigate to different pages to view various articles of "news". You can find the live version here: https://nc-news-newest.netlify.app as well as a link to my API on github: https://github.com/ptreedev/nc-news-api
 
-Currently, two official plugins are available:
+## Running the project locally
 
-- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
-- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+If you wish to run the project on your local machine follow the steps below to get started:
+
+1. First clone the project from github onto your local machine (instructions can be found at https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
```

Comment:
This isnt needed, you can assuming anyone wanting to clone this down will know how to